collect_app/src/main/java/org/odk/collect/android/map/WebMapService.java

```diff
@@ -0,0 +1,65 @@
+package org.odk.collect.android.map;
+
+import org.odk.collect.android.application.Collect;
+import org.osmdroid.tileprovider.MapTile;
+import org.osmdroid.tileprovider.tilesource.OnlineTileSourceBase;
+
+import java.io.Serializable;
+
+/**
+ * A serializable definition of a Web Map Service in terms of its URL structure
+ * and the parameters for fetching tiles from it.
+ */
```

Comment:
This is basically the Serializable version of what used to be the OnlineTileSourceBase objects created by TileSourceFactory.